Create configuration documents for Helm Broker (#4105)
kyma-project/kyma: docs/helm-broker/05-02-etcd-stateful.md

---
title: Etcd-stateful sub-chart
type: Configuration
---

To configure the Etcd-stateful sub-chart, override the default values of its `values.yaml` file. This document describes parameters that you can configure.

>**TIP:** To learn more about how to use overrides in Kyma, see the following documents:
>* [Helm overrides for Kyma installation](/root/kyma/#configuration-helm-overrides-for-kyma-installation)
>* [Sub-charts overrides](/root/kyma/#configuration-helm-overrides-for-kyma-installation-sub-chart-overrides)

## Configurable parameters

This table lists the configurable parameters, their descriptions, and default values:

| Parameter | Description | Default value |
|-----------|-------------|---------------|

> | **etcd.secure** | Specifies whether TLS can be used. | `false` |

Reviewer: ```suggestion
| **etcd.secure** | Indicates if TLS is used. | `false` |
```

| **etcd.resources.limits.cpu** | Defines limits for CPU resources. | `200m` |
| **etcd.resources.limits.memory** | Defines limits for memory resources. | `256Mi` |
| **etcd.resources.requests.cpu** | Defines requests for CPU resources. | `50m` |
| **etcd.resources.requests.memory** | Defines requests for memory resources. | `64Mi` |

> | **replicaCount** | Defines the number of etcd Pods in a cluster. | `1` |

Reviewer: ```suggestion
| **replicaCount** | Defines the size of the etcd cluster. | `1` |
```
